src/backend/gpopt/gpopt.mk

```diff
@@ -1,4 +1,3 @@
-override CPPFLAGS := -std=c++98 $(CPPFLAGS)
```

Comment:
Why did you remove this line?

Reply:
oops, that was not supposed to be commited

Reply:
fixed